### DISCUSSION TITLE
[Windowing] Making SDL default on windows ?

### DISCUSSION THREAD

**Discussion by Eideren** [stride3d/stride#971] [General]
Given the extensive additions the vvvv team pushed to our SDL windows should we set the default on Windows to that instead of winforms ?
Could we remove winforms and the build logic associated to it from the engine ? 
Granted it mostly works right now so there's not a whole lot of reason to do so but I'm all for reducing maintenance, engine and build complexity.

**Comment by herocrab** [comment]
From an issue reported by SoulRider today there are bugs with minimizing and maximizing full screen using graphics device manager. I think that's still the way identified in the docs to work with this. If these issues are resolved then we should make SDL default and update the stride docs. The engine is tough to pick up when we have small legacy half-broken things lying around.

**Comment by tebjan** [comment]
Yes to all points, good initiative!
